src/EditorFeatures/CSharpTest/Completion/CompletionProviders/Snippets/CSharpSnippetCompletionProviderTests.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Completion.CompletionProviders.Snippets;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Completion.CompletionProviders.Snippets
+{
+    public class CSharpSnippetCompletionProviderTests : AbstractCSharpCompletionProviderTests
```

Comment:
I'd like to see more test cases. Off the top of my head, these cases aren't covered (and would either be positive or negative)

1. Switch expression
2. Switch Statement
3. Single line lambda `DoSomething(x => $$)` 
4. Inside string
5. Argument location
6. Object initializer
7. Parameters 
8. Record declaration
9. file scoped namespace (you have block namespace covered)
10. Variable assignment/declaration

Reply:
done